### PR TITLE
fix: settings pane close icon hover

### LINKED ISSUE #19198: [Bug]: The "Close Settings Panel" icon does not have a hover state
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

No hover state for the `Close Settings Panel` is abserved.
Screenshot 2022-12-26 at 11 39 05 AM

### Steps To Reproduce

1. Go to any application.
2. Click on the `App Settings` button.
3. Hover over the `Close Settings Panel`.

### Public Sample App

_No response_

### Issue video log

_No response_

### Version

Cloud

### PR DESCRIPTION
**Pull request by eco-monk**
## Description

Add a hover state to the close icon.
image

Fixes #19198

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- Manual

### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: app/client/src/pages/Editor/AppSettingsPane/PaneHeader.tsx [hunk 1]
```diff
@@ -12,14 +12,24 @@ import styled from "styled-components";
...
+  padding: 10px 0px 10px;
   border-bottom: 1px solid ${Colors.GRAY_300};
+  margin-bottom: 0;
 `;
 
 const StyledText = styled.div`
   font-size: 16px;
 `;
 
+const StyledIcon = styled(Icon)`
```
Comment (hetunandu) [inline app/client/src/pages/Editor/AppSettingsPane/PaneHeader.tsx:24]: Does the Icon not have an hover state? This should come from the Design System

Comment (eco-monk) [reply app/client/src/pages/Editor/AppSettingsPane/PaneHeader.tsx:24]: cc: @RoopKrrish9696

**Review by hetunandu — APPROVED** [review]

**Comment by Richarex** [comment]
This issue is fixed verified it on the DP url.

**Pull request merged**